runtime/util/thrinfo.c

```diff
@@ -287,7 +287,17 @@ getVMThreadStateHelper(J9VMThread *targetThread,
 						 */
 					}
 				}
-	
+
+				/* for thread with a virtual thread mount that is waiting on a lock,
+				 * we should return the virtualthreadobject as lock object/owner */
+#if JAVA_SPEC_VERSION >= 19
```

Comment:
```suggestion
#if JAVA_SPEC_VERSION >= 19
				/* For thread with a VirtualThread mounted that is waiting on a lock,
				 * we should return the VirtualThread as lock object/owner */
```
format/grammar nitpick